utils/build/docker/php/common/php.ini

```diff
@@ -27,7 +27,7 @@ datadog.appsec.log_file=/var/log/system-tests/appsec.log
 datadog.appsec.helper_path=/usr/local/lib/libddappsec-helper.so
 datadog.appsec.helper_socket_path=/tmp/ddappsec.sock
 datadog.appsec.helper_lock_path=/tmp/ddappsec.lock
-; legacy
+datadog.appsec.helper_log_level=info
```

Comment:
System test environment should not have more logging level than customer do by default

Reply:
info is the default

Reply:
Ok I thought it was warning